Resources/public/js/views/fields/ez-time-editview.js

```diff
@@ -101,14 +101,17 @@ YUI.add('ez-time-editview', function (Y) {
         _variables: function () {
             var def = this.get('fieldDefinition'),
                 field = this.get('field'),
-                time = '';
+                time = '',
+                options;
 
             if ( field && field.fieldValue ) {
                 if (!this.get('supportsTimeInput') && !def.fieldSettings.useSeconds) {
-                    time =  Y.Date.format(new Date(field.fieldValue * 1000), {format:"%R"});
+                    options = {hour: 'numeric', minute: 'numeric', timeZone:'UTC'};
                 } else {
-                    time =  Y.Date.format(new Date(field.fieldValue * 1000), {format:"%T"});
+                    options = {hour: 'numeric', minute: 'numeric', second: 'numeric', timeZone:'UTC'};
                 }
+
+                time = new Date(field.fieldValue * 1000).toLocaleTimeString(undefined, options);
```

Comment:
unfortunately, the support for the parameters of `toLocaleTimeString` is rather low (Safari does not support that it seems plus several mobile agent), so we cannot use it